hack/images/kicbase.Dockerfile

```diff
@@ -16,12 +16,16 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     libglib2.0-0=2.62.1-1 \
     && rm /etc/crictl.yaml
 # install cri-o based on https://github.com/cri-o/cri-o/commit/96b0c34b31a9fc181e46d7d8e34fb8ee6c4dc4e1#diff-04c6e90faac2675aa89e2176d2eec7d8R128
+ENV CRIO_VERSION="1.17=1.17.3~2"
```

Comment:
Why do we need these exported in the runtime environment ? Are they used somewhere else ?

Reply:
I was inspired by the Skaffold dockerfile, which makes it really easy to upgrade versions by using variables:

https://github.com/GoogleContainerTools/skaffold/blob/master/deploy/skaffold/Dockerfile.deps#L17

But you're right that these don't need to be in the runtime env; it should be ARG instead of ENV